src/lib/sidebar/PipelineSchemeForm.svelte

```diff
@@ -3,157 +3,180 @@
   import {
     Checkbox,
     DefaultButton,
+    ErrorMessage,
+    NumberInput,
     Radio,
     SecondaryButton,
     TextArea,
+    TextInput,
   } from "lib/govuk";
   import { gjScheme } from "stores";
   import ATF4Type from "../forms/ATF4Type.svelte";
 
-  $gjScheme.pipeline ||= {
-    scheme_type: "",
-    status: "",
-    timescale: "",
-    atf4_lead_type: "",
-    scheme_description: "",
-    funding_source: "",
-    funded: false,
-  };
-  // Guaranteed to exist
-  let pipelineProps = $gjScheme.pipeline;
```

Comment:
Svelte footgun, oops! These weren't getting synced to local storage because of the indirection and weren't getting cleared properly

Reply:
I did find myself confused by this a couple of times, glad you tracked it down!